tests/publiccloud/run_ltp.pm

```diff
@@ -94,22 +91,28 @@ sub run {
     $self->{ltp_env} = $ltp_env;
 
     # Use lib/LTP/WhiteList module to exclude tests
-    if (get_var('LTP_KNOWN_ISSUES')) {
-        my $whitelist = LTP::WhiteList->new();
-        my $exclude = get_var('LTP_COMMAND_EXCLUDE', '');
-        my @skipped_tests = $whitelist->list_skipped_tests($ltp_env, get_required_var('LTP_COMMAND_FILE'));
-        if (@skipped_tests) {
-            $exclude .= '|' if (length($exclude) > 0);
-            $exclude .= '^(' . join('|', @skipped_tests) . ')$';
-            set_var('LTP_COMMAND_EXCLUDE', $exclude);
+    my $issues = get_var('LTP_KNOWN_ISSUES', '');
+    my $skip_tests;
+    if ($issues) {
+        my $whitelist = LTP::WhiteList->new($issues);
+        my @skipped = $whitelist->list_skipped_tests($ltp_env, $ltp_command);
+        if (@skipped) {
+            $skip_tests = '^(' . join("|", @skipped) . ')$';
+            record_info(
+                "Exclude",
+                "Excluding tests: $skip_tests",
+                result => 'softfail'
```

Comment:
why soft fail ? we just passing just FYI here and softfail meant to be used when there is some softfailure

Reply:
Moreover, please consider here this info is duplicated or excessively detailed, if we provide the below _Full exclude_ info.